nois-testnet-003/genesis.json

```diff
@@ -0,0 +1,329 @@
+          "account_number": "0",
+          "sequence": "0"
+        }
+      ]
+    },
+    "authz": {
+      "authorization": []
+    },
+    "bank": {
+      "params": {
+        "send_enabled": [],
+        "default_send_enabled": true
+      },
+      "balances": [
+        {
+          "address": "nois1rw7pvjluutkw9mk4qv3svyqtyfnz0dyzez2y9t",
+          "coins": [
+            {
+              "denom": "unois",
+              "amount": "50000000000000"
+            }
+          ]
+        },
+        {
+          "address": "nois1wdwucn7yvndrjzv9qqdt3pppcmw5xtx3er8zzk",
+          "coins": [
+            {
+              "denom": "unois",
+              "amount": "50000000000000"
+            }
+          ]
+        }
+      ],
+      "supply": [
+        {
+          "denom": "unois",
+          "amount": "100000000000000"
+        }
+      ],
+      "denom_metadata": []
+    },
+    "capability": {
+      "index": "1",
+      "owners": []
+    },
+    "crisis": {
+      "constant_fee": {
+        "denom": "unois",
+        "amount": "1000"
+      }
+    },
+    "distribution": {
+      "params": {
+        "community_tax": "0.020000000000000000",
+        "base_proposer_reward": "0.010000000000000000",
+        "bonus_proposer_reward": "0.040000000000000000",
+        "withdraw_addr_enabled": true
+      },
+      "fee_pool": {
+        "community_pool": []
+      },
+      "delegator_withdraw_infos": [],
+      "previous_proposer": "",
+      "outstanding_rewards": [],
+      "validator_accumulated_commissions": [],
+      "validator_historical_rewards": [],
+      "validator_current_rewards": [],
+      "delegator_starting_infos": [],
+      "validator_slash_events": []
+    },
+    "evidence": {
+      "evidence": []
+    },
+    "feegrant": {
+      "allowances": []
+    },
+    "genutil": {
+      "gen_txs": [
+        {
+          "body": {
+            "messages": [
+              {
+                "@type": "/cosmos.staking.v1beta1.MsgCreateValidator",
+                "description": {
+                  "moniker": "nois-validator",
+                  "identity": "",
+                  "website": "",
+                  "security_contact": "",
+                  "details": ""
+                },
+                "commission": {
+                  "rate": "0.100000000000000000",
+                  "max_rate": "0.200000000000000000",
+                  "max_change_rate": "0.010000000000000000"
+                },
+                "min_self_delegation": "1",
+                "delegator_address": "nois1wdwucn7yvndrjzv9qqdt3pppcmw5xtx3er8zzk",
+                "validator_address": "noisvaloper1wdwucn7yvndrjzv9qqdt3pppcmw5xtx3cr4rmz",
+                "pubkey": {
+                  "@type": "/cosmos.crypto.ed25519.PubKey",
+                  "key": "h/U47bWxrowe93gwKK1Rkb66CjZdm0iVdPIsw/yKn5k="
+                },
+                "value": {
+                  "denom": "unois",
+                  "amount": "500000000000"
```

Comment:
Can we start this with 20% of supply and then re-delegate from here?

Reply:
Ah, okay. I got confused by the 2x50M values from above and did not count zeros here